acceptance-tests/src/test/scala/ch/renku/acceptancetests/workflows/Datasets.scala

```diff
@@ -19,8 +25,90 @@ trait Datasets {
     reload whenUserCannotSee datasetPageTitle
 
     Then(s"the user should see a link to the '$datasetName' dataset")
-    val datasetLink = projectPage.Datasets.DatasetsList.link(to = datasetName)(_: WebDriver)
+    val datasetLink = projectPage.Datasets.DatasetsList.link(to = datasetPage)(_: WebDriver)
     reload whenUserCannotSee datasetLink
-    verify userCanSee projectPage.Datasets.DatasetsList.link(to = datasetName)
+    verify userCanSee projectPage.Datasets.DatasetsList.link(to = datasetPage)
+  }
+
+  def `create a dataset`(datasetName: DatasetName)(implicit projectPage: ProjectPage): DatasetPage = {
+    import Modification._
+    val newDatasetName = datasets.DatasetName("new")
+    val newDatasetPage = DatasetPage(newDatasetName)
+    Given("the user is on the Datasets tab")
+    click on projectPage.Datasets.tab
+
+    When("the user clicks on the Add Dataset button")
+    click on projectPage.Datasets.addADatasetButton
+    verify that newDatasetPage.ModificationForm.formTitle contains "Add Dataset"
+
+    And(s"the user add the title '$datasetName' to the new dataset")
+    `changing its title`(to = datasetName.toString).execute(newDatasetPage)
+
+    And("the user saves the modification")
+    click on newDatasetPage.ModificationForm.datasetSubmitButton sleep (10 seconds)
+
+    val datasetPage = DatasetPage(datasetName)
+    Then("the user should see its newly created dataset and the project it belongs to")
+    verify browserAt datasetPage
+    verify that datasetPage.datasetTitle contains datasetName.value
+    val projectLink = datasetPage.ProjectsList.link(projectPage)(_: WebDriver)
+    reload whenUserCannotSee projectLink
+    verify userCanSee datasetPage.ProjectsList.link(projectPage)
```

Comment:
Do we need to verify the link presence again after the reloading passes?

Reply:
Yes I don't think we do. I will also remove it from verifyDatasetCreated fonction